module-1/lab-api-scavenger-game/your-code/challenge-1.ipynb

```diff
@@ -0,0 +1,1847 @@
+{
```

Comment:
Well done Alberto!! Your code works.  I would suggest you to normalize JSONs before storing info as pandas dataframe on further occasions.